tests/unit_tests/test_avatars.py

```diff
@@ -0,0 +1,111 @@
+import importlib
+import json
+import re
+import unittest
+from unittest.mock import Mock
+import datetime
+import os
+import tempfile, shutil
+
+import faf.db as db
+import api
+from api.user import User, UserGroup
+from api.error import ErrorCode
+from api.avatars import Avatar
+
+class AvatarTestCase(unittest.TestCase):
```

Comment:
Could we stick to the `py.test` suite format? Or is there a reason you're using `unittest` here?